ldm/dream/pngwriter.py

```diff
@@ -36,9 +36,9 @@ def unique_prefix(self):
     def save_image_and_prompt_to_png(self, image, dream_prompt, name, metadata=None):
         path = os.path.join(self.outdir, name)
         info = PngImagePlugin.PngInfo()
-        info.add_text('Dream', dream_prompt)
+        info.add_text('Dream', dream_prompt or '')
         if metadata: # TODO: merge command line app's method of writing metadata and always just write metadata
-          info.add_text('sd-metadata', json.dumps(metadata))
+          info.add_text('sd-metadata', metadata)
```

Comment:
Seems like there is some IDE config that sets 2 spaces. Everything else seems to have 4, so best go with that (again, formatter would fix trivially :) )

Reply:
Yah looks like my IDE was set at 2 spaces. Ran a formatter on the code.